.travis.yml

```diff
@@ -1,49 +1,99 @@
-sudo: required
-dist: xenial
-
 os: linux
+dist: xenial
+sudo: false
 
 language: c
 
+arch:
+  - amd64
+ #- arm64
```

Comment:
The luajit2-test-suite is disabled for now on arm64 since it currently has a few failing test cases.